### ISSUE TITLE
ibm_is_lb: Total provision time too long

### ISSUE THREAD

**Issue by sean-freeman** [IBM-Cloud/terraform-provider-ibm#5380]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform IBM Provider Version

N/A

### Affected Resource(s)

* [`ibm_is_lb`](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/is_lb)

### Terraform Configuration Files

```hcl
resource "ibm_is_lb" "example" {
  name    = "example-load-balancer"
  subnets = [ibm_is_subnet.example.id, ibm_is_subnet.example1.id]
}

resource "ibm_is_lb_pool" "example" {
  name           = "example-pool"
  lb             = ibm_is_lb.example.id
  algorithm      = "round_robin"
  protocol       = "http"
  health_delay   = 60
  health_retries = 5
  health_timeout = 30
  health_type    = "http"
  proxy_protocol = "v1"
}

resource "ibm_is_lb_pool_member" "example_member1" {
  lb             = ibm_is_lb.example.id
  pool           = element(split("/", ibm_is_lb_pool.example.id), 1)
  port           = 8080
  target_address = "127.0.0.1"
  weight         = 60
}

resource "ibm_is_lb_pool_member" "example_member2" {
  lb             = ibm_is_lb.example.id
  pool           = element(split("/", ibm_is_lb_pool.example.id), 1)
  port           = 8080
  target_address = "127.0.0.1"
  weight         = 60
}

resource "ibm_is_lb_listener" "example" {
  lb                         = ibm_is_lb.example.id
  port                       = "9080"
  protocol                   = "http"
  https_redirect_listener    = ibm_is_lb_listener.example.listener_id
  https_redirect_status_code = 301
  https_redirect_uri         = "/example?doc=get"
}

```

### Expected Behavior

Terraform Resource `ibm_is_lb` should follow API Specification and upon `create` allow data input via nested:
- pools
    - pool server members
- listeners 

Citation:
- Create https://cloud.ibm.com/apidocs/vpc/latest#create-load-balancer

### Actual Behavior

Terraform Resources are modular-only, there is no allowance for nested creation.

This means every end-user must use in sequence:
- `ibm_is_lb` _(5-10 minutes provision time)_
    - `ibm_is_lb_pool` _(~4 minutes provision time, as a new pool will cause update/scan of the LB instance)_
        - `ibm_is_lb_pool_member` for HA Pair Node A _(~4 minutes provision time, as a new pool will cause LB update/scan )_
        - `ibm_is_lb_pool_member` for HA Pair Node B _(~4 minutes provision time, as a new pool will cause LB update/scan)_
    - `ibm_is_lb_listener` _(~4 minutes provision time, as a new pool will cause update/scan of the LB instance)_

== 5 + 16 minutes approximately using Terraform versus 5 minutes from API/CLI/Web GUI, to create 1 listener (e.g. Port 443) and 1 pool (with 2 pool server members)

This is a compounding problem as there are very few cases that use such as simple Load Balancer configuration. A more reasonable expectation for an average setup would be:
- `ibm_is_lb` _(5-10 minutes provision time)_
    - 5x `ibm_is_lb_pool` _(~20 minutes provision time, as a new pool will cause update/scan of the LB instance)_
        - For each Pool, `ibm_is_lb_pool_member` for HA Pair Node A _(~20 minutes provision time, as a new pool will cause LB update/scan )_
        - For each Pool, `ibm_is_lb_pool_member` for HA Pair Node B _(~20 minutes provision time, as a new pool will cause LB update/scan)_
    - 5x `ibm_is_lb_listener` _(~20 minutes provision time, as a new pool will cause update/scan of the LB instance)_

== 5 + 80 minutes approximately using Terraform versus 5 minutes from API/CLI/Web GUI, to create 5 listener (e.g. Port 443) and 5 pools (each with 2 pool server members).

> TL;DR - An average Load Balancer setup will take ~80 minutes longer with Terraform than the API/CLI/Web GUI, as the Load Balancer configuration is amended one-by-one and causes exponential lag.

I can appreciate how the modular-only approach would be considered the correct approach, it is logical for Terraform purposes and there is no nested pools/members and listeners on the `update`[PATCH] API Endpoint.

A compromise needs to be found as this is far too long for execution of a Load Balancer setup. This may mean `ibm_is_lb` logic needs to be expanded to mask complexity and handle multiple API calls:

- IF Load Balancer does not exist, execute `create` as currently + call API with `pools` and `listeners`
- IF Load Balancer exists
    - remove `pools` and `listeners` input and confirm no update to make to the Resource, as currently
    - for each item in `pools`, call existing logic in `ibm_is_lb_pool` Terraform Resource
    - for each item in `pools.members`, call existing logic in `ibm_is_lb_pool_member` Terraform Resource
    - for each item in `listeners`, call existing logic in `ibm_is_lb_listener`

**Comment by sean-freeman** [comment]
@hkantare Please help advocate for development effort to address this very significant problem. See TL;DR below:

> TL;DR - An average Load Balancer setup will take ~80 minutes longer with Terraform than the API/CLI/Web GUI, as the Load Balancer configuration is amended one-by-one (each time the LB restarts) and causes exponential increase in lag.

**Comment by hkantare** [comment]
@ujjwal-ibm Can you look into this issue

**Comment by ujjwal-ibm** [comment]
looking into it

**Comment by sean-freeman** [comment]
@hkantare @ujjwal-ibm  Please also has a major impact on [Red Hat OpenShift Container Platform (OCP) self-managed setup on IBM Cloud, using Installer-Provisioned Installation (IPI)](https://docs.openshift.com/container-platform/4.16/installing/installing_ibm_cloud_public/installing-ibm-cloud-vpc.html) executed by the [Red Hat OpenShift Installer](https://github.com/openshift/installer) _(which uses Terraform to provision resources as part of the IPI setup procedure)_

The runtime was `49m 36s` for Red Hat OCP Self-Managed IPI using the Red Hat OpenShift Installer

The total time was `24m 16s` for Red Hat OCP Self-Managed IPI to create/configure a very simple Load Balancer setup (which attempts to workaround delays by adding pool members last) is shown below including stdout

This equates to `48.92% of runtime for creating Load Balancer resources` during the runtime for Red Hat OCP Self-Managed IPI using the Red Hat OpenShift Installer.  This is a significant impact on the installer runtime.


### Timing

- 1x `ibm_is_lb` _(6m8s)_
    - 2x `ibm_is_lb_pool` _(1m7s)_
    - 5x `ibm_is_lb_listener` _(2m49s)_

- 1x `ibm_is_lb`
    - 2x `ibm_is_lb_pool`
        - Per Pool, 1x `ibm_is_lb_pool_member` _(3m28s)_
        - Per Pool, 3x `ibm_is_lb_pool_member` _(10m34s)_


### Resources, grouped by those which ran in parallel:

```
# Task 1 - 6m8s

DEBUG module.vpc.ibm_is_lb.kubernetes_api_private: Creation complete after 6m8s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c]


# Task 2 - 1m17s

DEBUG module.vpc.ibm_is_lb_pool.machine_config: Creation complete after 38s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-f7184f9e-613e-4052-8112-a8e3b7fc3903]
DEBUG module.vpc.ibm_is_lb_pool.kubernetes_api_private: Creation complete after 1m17s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-ac37f5bb-174f-4161-bab1-b06b0ed908ba]


# Task 3 - 2m49s

DEBUG module.vpc.ibm_is_lb_listener.machine_config: Creation complete after 2m4s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-c969b6d3-05d5-4c37-8953-14152ba54aed]
DEBUG module.vpc.ibm_is_lb_listener.kubernetes_api_private: Creation complete after 2m49s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-4c551850-9075-4a73-8db8-0a5bed2ef251]


# Task 4 - 3m28s

DEBUG ibm_is_lb_pool_member.machine_config: Creation complete after 1m46s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-f7184f9e-613e-4052-8112-a8e3b7fc3903/r006-a77eb4fb-6883-43e2-8e3a-6e0c06a4d649]
DEBUG ibm_is_lb_pool_member.kubernetes_api_private: Creation complete after 3m28s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-ac37f5bb-174f-4161-bab1-b06b0ed908ba/r006-b2d8cbe9-531d-49c4-9610-6802d00d3418]


# Task 5 (loop, LB must update one at a time) - 10m34s

DEBUG ibm_is_lb_pool_member.machine_config[2]: Creation complete after 3m22s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-f7184f9e-613e-4052-8112-a8e3b7fc3903/r006-33ee227b-3744-46c1-ae64-e951803fd5fc]
DEBUG ibm_is_lb_pool_member.machine_config[1]: Creation complete after 5m7s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-f7184f9e-613e-4052-8112-a8e3b7fc3903/r006-9dec705c-24b3-4791-9aa9-3e0446b5f85b]
DEBUG ibm_is_lb_pool_member.machine_config[0]: Creation complete after 8m49s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-f7184f9e-613e-4052-8112-a8e3b7fc3903/r006-af0206ae-14eb-4a12-8273-189c88344970]

DEBUG ibm_is_lb_pool_member.kubernetes_api_private[2]: Creation complete after 1m30s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-ac37f5bb-174f-4161-bab1-b06b0ed908ba/r006-8f10edda-af0d-41d0-beec-48e1f61cb688]
DEBUG ibm_is_lb_pool_member.kubernetes_api_private[1]: Creation complete after 6m58s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-ac37f5bb-174f-4161-bab1-b06b0ed908ba/r006-5eb531e6-ff4a-41ce-8c6a-1887bda81106]
DEBUG ibm_is_lb_pool_member.kubernetes_api_private[0]: Creation complete after 10m34s [id=r006-df3031fa-d67d-42ca-8ad3-d39cba34688c/r006-ac37f5bb-174f-4161-bab1-b06b0ed908ba/r006-cc59fcde-a48c-4c25-aecb-5ae50bb12063]


# Total time =
# 24m 16s during Red Hat OCP Self-Managed IPI to create/configure Load Balancers
```

**Comment by ujjwal-ibm** [comment]
hi @sean-freeman We are looking to optimise pool and pool member. Currently the fix is being tested

**Comment by ujjwal-ibm** [comment]
```
resource "ibm_is_lb" "testacc_LB" {
  name = "${var.name}-lb"
  subnets = ["${var.subnet_id}"]
}
resource "ibm_is_lb_pool" "testacc_lb_pool" {
  count   = 5
  name = "${var.name}-lb-pool-${count.index}"
  lb = "${ibm_is_lb.testacc_LB.id}"
  algorithm = "round_robin"
  protocol = "http"
  health_delay= 45
  health_retries = 5
  health_timeout = 30
  health_type = "tcp"
}
resource "ibm_is_lb_pool_member" "testacc_lb_mem_wgt" {
  count   = 5
  lb = "${ibm_is_lb.testacc_LB.id}"
  pool = "${element(split("/",ibm_is_lb_pool.testacc_lb_pool[count.index].id),1)}"
  port 	=	"5005"
  target_address = "0.0.0.0"
}

```


```
Plan: 11 to add, 0 to change, 0 to destroy.
ibm_is_lb.testacc_LB: Creating...
ibm_is_lb.testacc_LB: Still creating... [10s elapsed]
ibm_is_lb.testacc_LB: Still creating... [20s elapsed]
ibm_is_lb.testacc_LB: Still creating... [30s elapsed]
ibm_is_lb.testacc_LB: Still creating... [40s elapsed]
ibm_is_lb.testacc_LB: Still creating... [50s elapsed]
ibm_is_lb.testacc_LB: Still creating... [1m0s elapsed]
ibm_is_lb.testacc_LB: Still creating... [1m10s elapsed]
ibm_is_lb.testacc_LB: Still creating... [1m20s elapsed]
ibm_is_lb.testacc_LB: Still creating... [1m30s elapsed]
ibm_is_lb.testacc_LB: Still creating... [1m40s elapsed]
ibm_is_lb.testacc_LB: Still creating... [1m50s elapsed]
ibm_is_lb.testacc_LB: Still creating... [2m0s elapsed]
ibm_is_lb.testacc_LB: Still creating... [2m10s elapsed]
ibm_is_lb.testacc_LB: Still creating... [2m20s elapsed]
ibm_is_lb.testacc_LB: Still creating... [2m30s elapsed]
ibm_is_lb.testacc_LB: Still creating... [2m40s elapsed]
ibm_is_lb.testacc_LB: Still creating... [2m50s elapsed]
ibm_is_lb.testacc_LB: Still creating... [3m0s elapsed]
ibm_is_lb.testacc_LB: Still creating... [3m10s elapsed]
ibm_is_lb.testacc_LB: Still creating... [3m20s elapsed]
ibm_is_lb.testacc_LB: Still creating... [3m30s elapsed]
ibm_is_lb.testacc_LB: Still creating... [3m40s elapsed]
ibm_is_lb.testacc_LB: Still creating... [3m50s elapsed]
ibm_is_lb.testacc_LB: Still creating... [4m0s elapsed]
ibm_is_lb.testacc_LB: Still creating... [4m10s elapsed]
ibm_is_lb.testacc_LB: Still creating... [4m20s elapsed]
ibm_is_lb.testacc_LB: Still creating... [4m30s elapsed]
ibm_is_lb.testacc_LB: Still creating... [4m40s elapsed]
ibm_is_lb.testacc_LB: Still creating... [4m50s elapsed]
ibm_is_lb.testacc_LB: Still creating... [5m0s elapsed]
ibm_is_lb.testacc_LB: Still creating... [5m10s elapsed]
ibm_is_lb.testacc_LB: Still creating... [5m20s elapsed]
ibm_is_lb.testacc_LB: Still creating... [5m30s elapsed]
ibm_is_lb.testacc_LB: Still creating... [5m40s elapsed]
ibm_is_lb.testacc_LB: Still creating... [5m50s elapsed]
ibm_is_lb.testacc_LB: Still creating... [6m0s elapsed]
ibm_is_lb.testacc_LB: Still creating... [6m10s elapsed]
ibm_is_lb.testacc_LB: Still creating... [6m20s elapsed]
ibm_is_lb.testacc_LB: Still creating... [6m30s elapsed]
ibm_is_lb.testacc_LB: Still creating... [6m40s elapsed]
ibm_is_lb.testacc_LB: Creation complete after 6m45s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5]
ibm_is_lb_pool.testacc_lb_pool[2]: Creating...
ibm_is_lb_pool.testacc_lb_pool[0]: Creating...
ibm_is_lb_pool.testacc_lb_pool[3]: Creating...
ibm_is_lb_pool.testacc_lb_pool[1]: Creating...
ibm_is_lb_pool.testacc_lb_pool[4]: Creating...
ibm_is_lb_pool.testacc_lb_pool[0]: Still creating... [10s elapsed]
ibm_is_lb_pool.testacc_lb_pool[4]: Still creating... [10s elapsed]
ibm_is_lb_pool.testacc_lb_pool[3]: Still creating... [10s elapsed]
ibm_is_lb_pool.testacc_lb_pool[2]: Still creating... [10s elapsed]
ibm_is_lb_pool.testacc_lb_pool[1]: Still creating... [10s elapsed]
ibm_is_lb_pool.testacc_lb_pool[1]: Still creating... [20s elapsed]
ibm_is_lb_pool.testacc_lb_pool[2]: Still creating... [20s elapsed]
ibm_is_lb_pool.testacc_lb_pool[4]: Still creating... [20s elapsed]
ibm_is_lb_pool.testacc_lb_pool[0]: Still creating... [20s elapsed]
ibm_is_lb_pool.testacc_lb_pool[3]: Still creating... [20s elapsed]
ibm_is_lb_pool.testacc_lb_pool[0]: Creation complete after 26s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-a455f407-b48a-42d6-b4f6-fe4d0585102a]
ibm_is_lb_pool.testacc_lb_pool[4]: Creation complete after 26s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-e0fa7453-c02e-4290-a64c-8ffb9bc446c6]
ibm_is_lb_pool.testacc_lb_pool[1]: Creation complete after 26s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-e050d638-c15f-4d3c-8ed1-ef95a227230c]
ibm_is_lb_pool.testacc_lb_pool[3]: Creation complete after 27s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-1f34d62a-afc1-4d2f-b859-c11e6baedded]
ibm_is_lb_pool.testacc_lb_pool[2]: Still creating... [30s elapsed]
ibm_is_lb_pool.testacc_lb_pool[2]: Creation complete after 30s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-f8e20051-663d-41cc-a926-ae4b61a49805]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Creating...
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Creating...
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Creating...
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Creating...
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Creating...
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [1m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [1m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [1m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [1m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [1m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [1m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [1m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [1m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [1m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [1m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [1m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [1m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [1m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [1m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [1m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [1m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [1m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [1m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [1m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Still creating... [1m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[1]: Creation complete after 1m30s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-e050d638-c15f-4d3c-8ed1-ef95a227230c/r006-f64dcf49-5542-4c2c-aacf-e3b4a0022057]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [1m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [1m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [1m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [1m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [1m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [1m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [1m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [1m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [2m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [2m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [2m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [2m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [2m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [2m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [2m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [2m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [2m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [2m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [2m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [2m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [2m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [2m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [2m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [2m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [2m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [2m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [2m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [2m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [2m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [2m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [2m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [2m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [3m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [3m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [3m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [3m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [3m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Still creating... [3m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [3m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [3m10s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[0]: Creation complete after 3m11s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-a455f407-b48a-42d6-b4f6-fe4d0585102a/r006-3ad48206-3064-4a44-8ad9-ae6e7c8127c5]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [3m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [3m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [3m20s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [3m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [3m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [3m30s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [3m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [3m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [3m40s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [3m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [3m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [3m50s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [4m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [4m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [4m0s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [8m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [8m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [8m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [8m35s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [8m35s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [8m35s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [8m45s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Still creating... [8m45s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [8m45s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[3]: Creation complete after 8m52s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-1f34d62a-afc1-4d2f-b859-c11e6baedded/r006-f93dd4e4-791d-43a4-aecc-2c260a25a73e]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [8m55s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [8m55s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [9m5s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [9m5s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [9m15s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [9m15s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [9m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [9m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [9m35s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [9m35s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [9m45s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [9m45s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [9m55s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [9m55s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [10m5s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [10m5s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [10m15s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [10m15s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [10m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Still creating... [10m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[2]: Creation complete after 10m25s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-f8e20051-663d-41cc-a926-ae4b61a49805/r006-7b2442d1-eddb-4f6f-9b05-f1acb2651334]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [10m35s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [10m45s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [10m55s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [11m5s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [11m15s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [11m25s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [11m35s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Still creating... [11m45s elapsed]
ibm_is_lb_pool_member.testacc_lb_mem_wgt[4]: Creation complete after 11m49s [id=r006-326a82e4-5380-4e89-8f2c-87d78bab5bb5/r006-e0fa7453-c02e-4290-a64c-8ffb9bc446c6/r006-93d0f621-f672-4d37-9ec4-3418ef2c6667]

Apply complete! Resources: 11 added, 0 changed, 0 destroyed.

```
